src/gap.c

```diff
@@ -1159,9 +1156,9 @@ Obj FuncMASTER_POINTER_NUMBER(Obj self, Obj o)
     }
 #ifdef USE_GASMAN
     if ((void **) o >= (void **) MptrBags && (void **) o < (void **) MptrEndBags) {
-        return INTOBJ_INT( ((void **) o - (void **) MptrBags) + 1 );
+        return ObjInt_UInt(((void **)o - (void **)MptrBags) + 1);
     } else {
-        return INTOBJ_INT( 0 );
+        return ObjInt_UInt(0);
```

Comment:
I really dislike this particular change: `INTOBJ_INT(0)` is a compile time constant, and I deliberately use that all over the code.